.github/workflows/kind.yaml

```diff
@@ -135,6 +208,16 @@ jobs:
           cilium sysdump --output-filename cilium-sysdump-out --hubble-flows-count 10000
         shell: bash {0} # Disable default fail-fast behaviour so that all commands run independently
 
+      # This needs to be in sync with the flag --nodes-without-cilium below
+      # during installation.
```

Comment:
Actually, no need for this comment because `NODES_WITHOUT_CILIUM` will keep us in sync.